src/index.html

```diff
@@ -6,8 +6,42 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+
+      <img class="imac" src="images/imac.jpeg"
+              alt="imacphoto">
+
+
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md):
3. [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

Reply:
Pay attention to this comment. To point 3 from checklist